### ISSUE TITLE
GWT Designer fails to open after update (Windows)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tsaiyilien/google-plugin-for-eclipse#104]
```
[apologies if this was already issued - I only found a similar one for linux...]

What steps will reproduce the problem?
1. Install Eclipse for Java Devs fresh from scratch.
2. Install Google Plugin for Eclipse fresh from scratch via update-link 
(http://dl.google.com/eclipse/plugin/4.2) (I checked all 5 items as shown in 
step 2 here: https://developers.google.com/eclipse/docs/install-eclipse-4.2)
3. Create a new WebApp Project including sample code
4. Try to open the main class in GWT editor

What is the expected output? What do you see instead?
I expect to see the GWT editor. Instead it says, it didn't work: "Plug-in 
org.eclipse.wb.core.ui was unable to load class 
org.eclipse.wb.internal.core.editor.multi.DesignerEditor." (After installing 
the google plugin (or updating, which has the same effect) even swing-apps 
can't be build using window builder, which worked before.)
In eclipse 3.7 it says : "java.lang.ClassNotFoundException: 
org.eclipse.wb.internal.core.editor.multi.DesignerEditor"


What version of the product are you using? On what operating system?
OS: Win7 64
The procedure is reproduceable on eclipse 3.7 and 4.2 (both latest stable 
versions: Juno Service Release 1 20120920-0800 AND Indigo Service Release 2 
20120216-1857)
Google Plugin v. 3.2.0
GWT Designer v. 3.1.2


Please provide any additional information below.
Unchecking GWT Core on installation doesn't solve the problem (it says it gets 
updated otherwise).

I have an installation on another machine, which I could transfer to this one: 
everything works fine, an update kills it, too. There I have Google Plugin 
3.1.3 and GWT 2.6.0

```

Original issue reported on code.google.com by `i...@fabian-schink.de` on 19 Feb 2013 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
Ok, we will test this on Windows and try to reproduce it. 
```

Original comment by `rdayal@google.com` on 20 Feb 2013 at 2:47
- Changed state: **Accepted**
- Added labels: **Release-3.2.1**

**Comment by GoogleCodeExporter** [comment]
```
Even im facing a similar issue.. Please check!

java.lang.ClassNotFoundException: 
org.eclipse.wb.internal.core.editor.multi.DesignerEditor
    at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:513)
    at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
    at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
    at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at org.eclipse.osgi.internal.loader.BundleLoader.loadClass(BundleLoader.java:345)
    at org.eclipse.osgi.framework.internal.core.BundleHost.loadClass(BundleHost.java:229)
    at org.eclipse.osgi.framework.internal.core.AbstractBundle.loadClass(AbstractBundle.java:1207)
    at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.createExecutableExtension(RegistryStrategyOSGI.java:174)
    at org.eclipse.core.internal.registry.ExtensionRegistry.createExecutableExtension(ExtensionRegistry.java:905)
    at org.eclipse.core.internal.registry.ConfigurationElement.createExecutableExtension(ConfigurationElement.java:243)
    at org.eclipse.core.internal.registry.ConfigurationElementHandle.createExecutableExtension(ConfigurationElementHandle.java:55)
    at org.eclipse.ui.internal.WorkbenchPlugin.createExtension(WorkbenchPlugin.java:260)
    at org.eclipse.ui.internal.registry.EditorDescriptor.createEditor(EditorDescriptor.java:235)
    at org.eclipse.ui.internal.EditorManager.createPart(EditorManager.java:875)
    at org.eclipse.ui.internal.EditorReference.createPartHelper(EditorReference.java:609)
    at org.eclipse.ui.internal.EditorReference.createPart(EditorReference.java:465)
    at org.eclipse.ui.internal.WorkbenchPartReference.getPart(WorkbenchPartReference.java:595)
    at org.eclipse.ui.internal.EditorReference.getEditor(EditorReference.java:289)
    at org.eclipse.ui.internal.WorkbenchPage.busyOpenEditorBatched(WorkbenchPage.java:2945)
    at org.eclipse.ui.internal.WorkbenchPage.busyOpenEditor(WorkbenchPage.java:2850)
    at org.eclipse.ui.internal.WorkbenchPage.access$11(WorkbenchPage.java:2842)
    at org.eclipse.ui.internal.WorkbenchPage$10.run(WorkbenchPage.java:2793)
    at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
    at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:2789)
    at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:2773)
    at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:2756)
    at org.eclipse.wb.internal.core.wizards.DesignerNewElementWizard$1.run(DesignerNewElementWizard.java:81)
    at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
    at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
    at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4140)
    at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3757)
    at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2701)
    at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2665)
    at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2499)
    at org.eclipse.ui.internal.Workbench$7.run(Workbench.java:679)
    at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
    at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:668)
    at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
    at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:123)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1410)

```

Original comment by `vamshi.p...@gmail.com` on 22 Feb 2013 at 4:39

Attachments:
- [ErrorLog.log](https://storage.googleapis.com/google-code-attachments/google-plugin-for-eclipse/issue-104/comment-2/ErrorLog.log)

**Comment by GoogleCodeExporter** [comment]
```
Has anybody figured out a work around?
```

Original comment by `shishirm...@gmail.com` on 24 Feb 2013 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
I was getting the same error message and was able to resolve it, although this 
was not on a fresh install like the OP, it's originally from the Android ADT. 
Somehow a 2.6 (stand-alone?) version of the GWT got installed early on, and 
removing it and updating the other components didn't seem to fix it.

To fix it, I uninstalled the GWT SDK, the three GWT Designer components 
(core/editor/gpe), and the three WindowBuilder components (core/css/xml). I 
restarted Eclipse when prompted, and then reinstalled everything but the NDK 
plugins from http://dl.google.com/eclipse/plugin/4.2 from inside Eclipse.

After creating a new web app project via the wizard, I had to open project 
settings and change the facet and the java compiler compliance version from 1.4 
to 1.6 to eliminate errors (though not sure this is related) but the GWT 
Designer then opened in design view without issue.
```

Original comment by `chach.fa...@gmail.com` on 24 Feb 2013 at 1:54

**Comment by GoogleCodeExporter** [comment]
```
Thanks chach, I also installed Eclipse for ADT, then modified for Sencha and 
now adding GWT had same error.

I followed your 2nd paragraph and all was good. Have not gotten into/looked at 
compiler stuff yet, don't know if I need it.

Thanks!
```

Original comment by `byrd.d...@gmail.com` on 1 Mar 2013 at 5:20

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot, and I also have same error.
I try to it on Ubuntu12.10 and Win8. There is a same problem.
I just install Eclipse IDE for Java EE Developers, and add plugin of Google via 
link(http://dl.google.com/eclipse/plugin/4.2).
The most of functions of this plugin work well, but WindowBuilder can't work. 
And WindowBuilder is not appear in "New".
I have installed eclipse of other version, and it don't have this problem. 
```

Original comment by `wangjunz...@gmail.com` on 4 Mar 2013 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
@wangjunzhe4444: actually WindowBuilder wizards and other UI contributions 
should not appear when you install GPE with GWT Designer. GPE embeds 
WindowBuilder as internal set of plugins and exposes only GWT Designer editor - 
no New wizards, no toolbar actions.

It seems that problem is not with what we have in build now, but with updating.
Basically you have to uninstall old version and install new one.
It may be caused by restructuring WindowBuilder done do extract support for 
Java into separate plugin.
Or may be some caching done by Eclipse itself.
Do you run Eclipse with -clean command line argument?

```

Original comment by `scheglov@google.com` on 4 Mar 2013 at 8:24

**Comment by GoogleCodeExporter** [comment]
```
I've installed GPE + SDKs + GWT Designer from 
http://dl.google.com/eclipse/plugin/4.2 into Eclipse SDK 4.2.1 and it works 
just fine. So, this makes me even more confident that current version itself is 
OK.
```

Original comment by `scheglov@google.com` on 4 Mar 2013 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
I just tried the setup from Feb. 19th, which I described originally (and which 
was a fresh installation, so I don't believe in caching-problems). It still 
didn't work.

BUT: I checked for updates and was provided with an update to     
"Eclipse IDE for Java Developers    1.5.2.20130211-1820"
which I installed.
After that, everything works fine now.

2nd "BUT": The above mentioned 2nd test concerning eclipse 3.7 still won't 
work. There are no updates available (which is no surprise) and -clean has no 
effect either.
```

Original comment by `i...@fabian-schink.de` on 4 Mar 2013 at 8:53

**Comment by GoogleCodeExporter** [comment]
```
OK, I can reproduce this when I install GPE on Eclipse for Java 4.2.1.

It seems that problem is that org.eclipse.wb.core.ui_1.5.1.r42x201209161116.jar 
was not updated and it expects that DesignerEditor class is in wb.core plugin. 
But this class was moved to wb.core.java and wb.core.ui plugin was updated. But 
by some reason this updated plugin was not picked up by GPE install.

It may be because GPE feature should not require also 
org.eclipse.wb.core.java.feature, something missed during Java plugin 
extraction. I will check this out.
```

Original comment by `scheglov@google.com` on 4 Mar 2013 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in next week's release. Apologies for the problem.

For a workaround, it should be sufficient to update WIndowBuilder manually, I 
think (via "Check for Updates").
```

Original comment by `rdayal@google.com` on 5 Mar 2013 at 6:20

**Comment by GoogleCodeExporter** [comment]
```
I installed eclipse-jee-juno-SR2-linux-gtk-x86_64.tar.gz then the plugin 
(http://dl.google.com/eclipse/plugin/4.2) , and then the designer 
(http://dl.google.com/eclipse/inst/d2gwt/latest/4.2).  I updated eclipse (2 
WindowBuilder items).  I closed eclipse then launched it with -clean.

I created the sample and opened ImageViewer.java.  I then re-opened it as 
Designer file.  Switching to Designer tab causes eclipse to crash and end.

This is Fedora 18 x64.  It fails in native code having to do with libwebkitgtk 
at org.eclipse.swt.widgets.Display.eventProc.  Please let me know anything that 
would be helpful.
```

Original comment by `rshender...@gmail.com` on 9 Mar 2013 at 3:20

Attachments:
- [hs_err_pid19005.log](https://storage.googleapis.com/google-code-attachments/google-plugin-for-eclipse/issue-104/comment-12/hs_err_pid19005.log)

**Comment by GoogleCodeExporter** [comment]
```
Same here on Ubuntu 12.10 with newest Oracle JDK 1.7:
1.) 
- Downloaded Eclipse 4.2.2 for java devs, fresh and clean
- Installed SWT Designer and SDKs by GPE from Google Repo for 4.2
- Created new gwt web project sample by wizard
- Try to open java file in client package via GWT Designer
- Crash to desktop with hs_err_pidXXX.log pointing to libwebkit

2.)
- Same like 1.) but tried Eclipse 3.7 instead
- Used Google repo for 3.7 to install GWT stuff
- Created sample and opened java file via gwt designer
- Results in "java.lang.ClassNotFoundException: 
org.eclipse.wb.internal.core.editor.multi.DesignerEditor" inside eclipse

I googled the whole day to get a fix for this. Can't even open the default 
sample project with a fresh clean installation on "normal" Ubuntu 12.10 
machine. I hope someone knows how to fix this. Tried a lot workarounds by using 
libxulrunner from lucid repo, set swt.default.browser to mozilla or webkit.
```

Original comment by `renegre...@googlemail.com` on 20 Mar 2013 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
What if you run Eclipse using this command line:
./eclipse -vmargs -Dorg.eclipse.swt.internal.gtk.cairoGraphics=false 
-Dorg.eclipse.swt.internal.gtk.useCairo=false

But crash with dump pointing to libwebkit seems to be a separated linux-only 
issue. 
```

Original comment by `alexande...@gmail.com` on 20 Mar 2013 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
Thank you for your answer,
I set up a fresh and clean eclipse 3.7 and 4.2. Used your arguments to start 
eclipse. Checked the configuration in the about dialog for the parameters to be 
set correctly.
But sadly there's no change:
- Eclipse 4.2 still crashes with hs_err_pid pointing to "C  
[libwebkitgtk-1.0.so.0+0x14245a0]  void 
WTF::freeOwnedGPtr<_GdkEvent>(_GdkEvent*)+0x15df0"
- Eclipse 3.7 still throws ClassNotFoundException

I attached both eclipse configurations, perhaps there's a problem with some 
versions?!

Thx for your effort!
```

Original comment by `renegre...@googlemail.com` on 21 Mar 2013 at 7:41

Attachments:
- [indigo_config.txt](https://storage.googleapis.com/google-code-attachments/google-plugin-for-eclipse/issue-104/comment-15/indigo_config.txt)
- [juno_config.txt](https://storage.googleapis.com/google-code-attachments/google-plugin-for-eclipse/issue-104/comment-15/juno_config.txt)

**Comment by GoogleCodeExporter** [comment]
```
I reverted xulrunner to v1.9.2 and tho it isn't perfect, it basically works.

firefox-3.6.10-1.fc14.x86_64                  Thu 21 Mar 2013 07:15:06 AM CDT
xulrunner-1.9.2.10-1.fc14.1.x86_64            Thu 21 Mar 2013 07:03:05 AM CDT
hunspell-en-0.20100322-6.fc14.noarch          Thu 21 Mar 2013 07:03:05 AM CDT
hunspell-1.2.12-1.fc14.x86_64                 Thu 21 Mar 2013 07:03:04 AM CDT
libpng12-1.2.50-2.fc18.x86_64                 Thu 21 Mar 2013 06:55:28 AM CDT
gcc-4.7.2-8.fc18.x86_64                       Tue 19 Mar 2013 07:34:45 AM CDT

```

Original comment by `rshender...@gmail.com` on 21 Mar 2013 at 12:41

**Comment by GoogleCodeExporter** [comment]
```
This sounds like some of the workarounds I found. I also tried an older version 
of xulrunner in combination with the appropriate libhunspell. This didn't work 
for me. I think  it's not a good idea to install old packages and old browser 
applications to make an actual IDE working. In my opinion there's is an 
incompatibility between the eclipse version and some OS related libraries like 
the JDK or libwebkit. So the solution should not be given by changing my 
working place. Furthermore the IDE should be up to date to work with an 
standard distro like ubuntu (As I mentioned before, this is just my opinion and 
not a bunch of criticism).
```

Original comment by `renegre...@googlemail.com` on 21 Mar 2013 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
Finally together with a colleague I found a solution which prevents the 
libwebkit crash and makes Kepler/Juno working for Ubuntu. Assuming that it is 
just OS configuration thing it should work for Fedora else.

Prerequisites:
- Oracle JDK 1.7
- Fresh and clean Eclipse Juno/Kepler (I used the JEE developer version).
- Install Google Eclipse Plugin, GWT Designer and GWT SDK via update site for 
Eclipse 4.2

Steps to make it work (Ubuntu):
1.) Download libhunspoell-1.2 deb package from the lucid repos 
(http://packages.ubuntu.com/lucid/amd64/libhunspell-1.2-0/download)
2. Download xulruner-1.9.2 deb package from the lucid repos 
(http://packages.ubuntu.com/lucid/amd64/xulrunner-1.9.2/download)
3. Install the packages via 'sudo dpkg -i *.deb' inside the folder you 
downloaded the  two deb packages. Make sure there are no other .deb packages!
4. Start terminal and type 'export GDK_NATIVE_WINDOWS=true'  I also found 
solutions where it is set to false. For me it doesn't matter if I set it to 
[TRUE|FALSE|<NOTHING>]. When checking the console, while eclipse is running and 
starting the designer, there is a print out "'GDK_NATIVE_WINDOWS' set, won't 
use WebKit.". It seems it only checks presence of this variable and not its 
value.
5. In the same terminal navigate to your eclipse and run it via ./eclipse 
command. This must happen inside the same temrinal, because the 
GDK_NATIVE_WINDOWS is only available inside this process.

I hope this works for others too. For me and me colleague it was the only 
solution getting the designer started.
```

Original comment by `renegre...@googlemail.com` on 22 Mar 2013 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
A small addition:
The designer starts but widgets are "invisible", respectively unstyled. To fix 
this I had to add a line at the end of the eclipse.ini: 
-Dorg.eclipse.swt.internal.gtk.cairoGraphics=false

```

Original comment by `renegre...@googlemail.com` on 22 Mar 2013 at 10:09

**Comment by GoogleCodeExporter** [comment]
```
The crash is related to WebKit libs installed in Ubuntu 12.10 
(libwebkitgtk-1.0.so.0.17.3). Previous versions of this lib (for example, 
libwebkitgtk-1.0.so.0.13.4) works fine. So installing old WebKit libs can help 
(though it is not recommended).
```

Original comment by `alexande...@gmail.com` on 22 Mar 2013 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
me too that error. please help me.

I installed gwt designer

ubuntu 12.10 /32bit but my laptop is 64 bit/
eclipse indigo /32bit/
gwt 2.5.1
app engine 1.7.5

java -version
java version "1.7.0_15"
OpenJDK Runtime Environment (IcedTea7 2.3.7) (7u15-2.3.7-0ubuntu1~12.10.1)
OpenJDK Server VM (build 23.7-b01, mixed mode)


I was creating new web application project. name:gwtTest, package: com and go 
to my project gwtTest -> src - > com.client gwtTest.java file right click open 
with -> GWT Designer or WindowBuilder editor but below error can't open 
designer editor.

Could not open the editor: The editor class could not be instantiated. This 
usually indicates a missing no-arg constructor or that the editor's class name 
was mistyped in plugin.xml.

error list:


java.lang.ClassNotFoundException: 
org.eclipse.wb.internal.core.editor.multi.DesignerEditor
    at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:513)
    at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
    at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
    at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:356)
    at org.eclipse.osgi.internal.loader.BundleLoader.loadClass(BundleLoader.java:345)
    at org.eclipse.osgi.framework.internal.core.BundleHost.loadClass(BundleHost.java:229)
    at org.eclipse.osgi.framework.internal.core.AbstractBundle.loadClass(AbstractBundle.java:1207)
    at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.createExecutableExtension(RegistryStrategyOSGI.java:174)
    at org.eclipse.core.internal.registry.ExtensionRegistry.createExecutableExtension(ExtensionRegistry.java:905)
    at org.eclipse.core.internal.registry.ConfigurationElement.createExecutableExtension(ConfigurationElement.java:243)
    at org.eclipse.core.internal.registry.ConfigurationElementHandle.createExecutableExtension(ConfigurationElementHandle.java:55)
    at org.eclipse.ui.internal.WorkbenchPlugin.createExtension(WorkbenchPlugin.java:260)
    at org.eclipse.ui.internal.registry.EditorDescriptor.createEditor(EditorDescriptor.java:235)
    at org.eclipse.ui.internal.EditorManager.createPart(EditorManager.java:875)
    at org.eclipse.ui.internal.EditorReference.createPartHelper(EditorReference.java:609)
    at org.eclipse.ui.internal.EditorReference.createPart(EditorReference.java:465)
    at org.eclipse.ui.internal.WorkbenchPartReference.getPart(WorkbenchPartReference.java:595)
    at org.eclipse.ui.internal.PartPane.setVisible(PartPane.java:313)
    at org.eclipse.ui.internal.presentations.PresentablePart.setVisible(PresentablePart.java:180)
    at org.eclipse.ui.internal.presentations.util.PresentablePartFolder.select(PresentablePartFolder.java:270)
    at org.eclipse.ui.internal.presentations.util.LeftToRightTabOrder.select(LeftToRightTabOrder.java:65)
    at org.eclipse.ui.internal.presentations.util.TabbedStackPresentation.selectPart(TabbedStackPresentation.java:473)
    at org.eclipse.ui.internal.PartStack.refreshPresentationSelection(PartStack.java:1245)
    at org.eclipse.ui.internal.PartStack.setSelection(PartStack.java:1198)
    at org.eclipse.ui.internal.PartStack.showPart(PartStack.java:1597)
    at org.eclipse.ui.internal.PartStack.add(PartStack.java:493)
    at org.eclipse.ui.internal.EditorStack.add(EditorStack.java:103)
    at org.eclipse.ui.internal.PartStack.add(PartStack.java:479)
    at org.eclipse.ui.internal.EditorStack.add(EditorStack.java:112)
    at org.eclipse.ui.internal.EditorSashContainer.addEditor(EditorSashContainer.java:63)
    at org.eclipse.ui.internal.EditorAreaHelper.addToLayout(EditorAreaHelper.java:225)
    at org.eclipse.ui.internal.EditorAreaHelper.addEditor(EditorAreaHelper.java:213)
    at org.eclipse.ui.internal.EditorManager.createEditorTab(EditorManager.java:808)
    at org.eclipse.ui.internal.EditorManager.openEditorFromDescriptor(EditorManager.java:707)
    at org.eclipse.ui.internal.EditorManager.openEditor(EditorManager.java:666)
    at org.eclipse.ui.internal.WorkbenchPage.busyOpenEditorBatched(WorkbenchPage.java:2942)
    at org.eclipse.ui.internal.WorkbenchPage.busyOpenEditor(WorkbenchPage.java:2850)
    at org.eclipse.ui.internal.WorkbenchPage.access$11(WorkbenchPage.java:2842)
    at org.eclipse.ui.internal.WorkbenchPage$10.run(WorkbenchPage.java:2793)
    at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
    at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:2789)
    at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:2773)
    at org.eclipse.ui.actions.OpenWithMenu.openEditor(OpenWithMenu.java:331)
    at org.eclipse.ui.actions.OpenWithMenu$2.handleEvent(OpenWithMenu.java:179)
    at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
    at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1258)
    at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:3588)
    at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3209)
    at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2701)
    at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2665)
    at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2499)
    at org.eclipse.ui.internal.Workbench$7.run(Workbench.java:679)
    at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
    at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:668)
    at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
    at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:123)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:601)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1410)

```

Original comment by `darama...@gmail.com` on 30 Mar 2013 at 1:09

**Comment by GoogleCodeExporter** [comment]
```
my related answer is below

http://stackoverflow.com/questions/15715079/gwt-designer-editor-cant-opened
```

Original comment by `darama...@gmail.com` on 30 Mar 2013 at 1:10

**Comment by GoogleCodeExporter** [comment]
```
on windows OS the same error any idea mymans?
```

Original comment by `darama...@gmail.com` on 31 Mar 2013 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
before install gwt designer it is OK, and after install gwt designer that error.
```

Original comment by `darama...@gmail.com` on 31 Mar 2013 at 8:12

**Comment by GoogleCodeExporter** [comment]
```
The only solution (post #18) for me was to use Eclipse Juno or Kepler. I was 
not able to fix the ClassNotFoundException on Indigo.
```

Original comment by `renegre...@googlemail.com` on 2 Apr 2013 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
thank you reply Rene. How can i install GWT desginer on eclipse Juno
```

Original comment by `darama...@gmail.com` on 2 Apr 2013 at 7:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdayal@google.com` on 3 Apr 2013 at 8:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
thank you for fix error. But i was update and reopen GWT Dedsigner editor
exit eclipse and following error what happened?

#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x7e3263a4, pid=19101, tid=3076409024
#
# JRE version: 7.0_17-b02
# Java VM: Java HotSpot(TM) Client VM (23.7-b01 mixed mode linux-x86 )
# Problematic frame:
# C  [libwebkitgtk-1.0.so.0+0x134f3a4]  void 
WTF::freeOwnedGPtr<_GdkEvent>(_GdkEvent*)+0x19ac4
#
# Failed to write core dump. Core dumps have been disabled. To enable core 
dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.sun.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  T H R E A D  ---------------

Current thread (0x085e9800):  JavaThread "main" [_thread_in_native, id=19101, 
stack(0xbf9eb000,0xbfa3b000)]

siginfo:si_signo=SIGSEGV: si_errno=0, si_code=1 (SEGV_MAPERR), 
si_addr=0x00000000

Registers:
EAX=0x00000000, EBX=0x7e7ef000, ECX=0x099c7230, EDX=0x00000000
ESP=0xbfa366ac, EBP=0x00000001, ESI=0xbfa366d0, EDI=0x085e3ff8
EIP=0x7e3263a4, EFLAGS=0x00210286, CR2=0x00000000

******************************************************

physical id : 0
siblings    : 2
core id     : 1
cpu cores   : 2
apicid      : 1
initial apicid  : 1
fdiv_bug    : no
hlt_bug     : no
f00f_bug    : no
coma_bug    : no
fpu     : yes
fpu_exception   : yes
cpuid level : 13
wp      : yes
flags       : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat 
pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe nx lm constant_tsc 
arch_perfmon pebs bts aperfmperf pni dtes64 monitor ds_cpl est tm2 ssse3 cx16 
xtpr pdcm xsave lahf_lm dtherm
bogomips    : 4189.31
clflush size    : 64
cache_alignment : 64
address sizes   : 36 bits physical, 48 bits virtual
power management:



Memory: 4k page, physical 4032972k(255304k free), swap 4093948k(3806292k free)

vm_info: Java HotSpot(TM) Client VM (23.7-b01) for linux-x86 JRE 
(1.7.0_17-b02), built on Mar  1 2013 03:10:07 by "java_re" with gcc 4.3.0 
20080428 (Red Hat 4.3.0-8)

time: Fri Apr  5 15:47:43 2013
elapsed time: 169 seconds

```

Original comment by `darama...@gmail.com` on 5 Apr 2013 at 7:59

**Comment by GoogleCodeExporter** [comment]
```
after 'check update', it works fine. Mine is 3.7(Indigo)
  WindowBuilder Core    1.5.2.r37x201303061457  
  WindowBuilder Core UI 1.5.2.r37x201303061516  
    WindowBuilder Java Core 1.5.2.r37x201303061505  
```

Original comment by `djx0...@gmail.com` on 10 Apr 2013 at 5:42

**Comment by GoogleCodeExporter** [comment]
```
"after 'check update', it works fine. Mine is 3.7(Indigo)
  WindowBuilder Core    1.5.2.r37x201303061457  "

Thanks, this worked for me too!
```

Original comment by `xenar...@gmail.com` on 22 Apr 2013 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
#29 djx0...@gmail.com
after 'check update', it works fine. Mine is 3.7(Indigo)
  WindowBuilder Core    1.5.2.r37x201303061457  
  WindowBuilder Core UI 1.5.2.r37x201303061516  
    WindowBuilder Java Core 1.5.2.r37x201303061505
<hr/>
Mine Eclipse is Juno 4.2 and it worked for me too, thanks a lot..

```

Original comment by `cihan.fo...@gmail.com` on 7 Jun 2013 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
 #18 renegre...@googlemail.com
This worked for me too. A *libwebkitgtk* issue. Thank you 'renegre...' for your 
time founding and sharing a solution.

```

Original comment by `gavra...@gmail.com` on 22 Jun 2013 at 2:42

**Comment by GoogleCodeExporter** [comment]
```
See also issue #83 
(https://code.google.com/p/google-plugin-for-eclipse/issues/detail?id=83).
```

Original comment by `nhcohen@google.com` on 24 Jun 2013 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
Issue 181 has been merged into this issue.
```

Original comment by `nhcohen@google.com` on 24 Jun 2013 at 2:54

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I solve the problem (Plug-in "org.eclipse.wb.core.ui" was unable to instantiate 
class "org.eclipse.wb.internal.core.editor.multi.DesignerEditor") updating the 
windowbuilder whit "Check for Updates", os: Debian, Eclipse 4.3
```

Original comment by `jesus...@gmail.com` on 29 Jul 2013 at 1:44
